setup.cfg

```diff
@@ -0,0 +1,13 @@
+[check-manifest]
+ignore =
+    */test
+    */test/*
+    .*
+    node_modules
+
+[pep257]
+ignore = D202
+explain = true
+
+[yapf]
+based_on_style = pep8
```

Comment:
My intentions weren't to open up a discussion on that topic, I rather just blatantly copy & pasted what he have on h with a few adaptions.

Shall I remove the yapf related lines?